synapse/tests/common.py

```diff
@@ -25,6 +28,14 @@
 
 class TooFewEvents(Exception): pass
 
+# Py2/3 SSL Exception Compat
+if s_compat.version >= (3, 0, 0):
```

Comment:
Yep - I didn't think it made sense to add syn exceptions because the exceptions occur outside of Synapse and even its dependencies. These exceptions by the http client making the http request against tornado. We should add syn exceptions if we make our own SynHTTPClient though.